### ISSUE TITLE
Where to set the labels used for training, such as 1/2, 1/4

### ISSUE THREAD

**Issue by acmername** [HUANGLIZI/LViT#40]
Looking forward to your reply
Thanks！

**Comment by HUANGLIZI** [comment]
Please use train_dataset, test_dataset = torch.utils.data.random_split(full_dataset, [train_size, test_size]) to split the dataset, and the random seed is the same as others.

**Comment by acmername** [comment]
Thank you for your timely recovery. Do you mean take half img and half labelcol from Train_Floder in the dataset file, subtract the corresponding data from train_text.xlsx and train?
